hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -229,21 +230,26 @@ N.B. NETCONSOLE blocks until the ethernet link is established or a timeout occur
 See https://wiki.archlinux.org/index.php/Netconsole
 ```
 src_port@src_ip/dev_name,dst_port@tgt_ip/tgt_mac
-E.g. 6665@169.254.1.1/eth0,6666@/
+E.g. 6665@169.254.1.1/,6666@/
```

Comment:
Does the removal of `eth0` here also mean that `dev_name` has a default value? (should be mentioned in the sentence below?)

Reply:
It's ignored